### DISCUSSION TITLE
Lyrics and Info album links don't work in local mode

### DISCUSSION THREAD

**Discussion by Garrido33** [rern/rAudio#185] [Fixed Issues]
@Rern 

In local mode (touchscreen), the Lyrics and Info album links don't work. No problem in remote mode.

Garrido33

**Comment by ceejayemm** [comment]
I can confirm this too. (DIY ROS + update 20210209)

Chris

**Comment by rern** [comment]
Lyrics - Fixed.
Album link - External link opens a new window which there's no back button in fullscreen mode so it's disabled.
